service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsSubmittedControllerTest.java

```diff
@@ -316,19 +314,6 @@ private CaseDetails createCase(Map<String, Object> data) {
             "caseUrl", "http://fake-url/cases/case-details/12345");
     }
 
-    private Map<String, Object> buildExpectedNotificationParams() {
-        C2UploadedTemplate c2UploadedTemplate = new C2UploadedTemplate();
```

Comment:
Removed this as I think we agreed that controllers didn't need fine level details of the parameters and instead could just use anyMap() so didn't see point in maintaining this as it's covered in unit tests - happy if someone disagrees.